index.html

```diff
@@ -2,220 +2,88 @@
+	<link href="https://fonts.googleapis.com/css?family=Quicksand:400,700" rel="stylesheet">
 	<link rel="icon" type="image/png" href="/favicon.png">
-	<script>
+	<!--<script>
 	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
 	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 	
 	  ga('create', 'UA-55967951-1', 'auto');
 	  ga('send', 'pageview');
-	</script>
+	</script>-->
 </head>
 <body>
-
-<header>
-	<div class="grid-container">
-	<a href="#" id="logo"><img src='/img/header_logo.png'></a>
-		<nav>
-			<a href="#about">About</a>
-			<a href="#ventures">Ventures</a>
-			<!--<a href="#team">Team</a>-->
-			<a href="#contact">Contact Us</a>
-		</nav>
-	</div>
-</header>
-
-<section id="intro">
-<div class="grid-container">
-
-	<h2 style="font-size:110px">the club for <strong style="color:#59686b">hackers</strong> <strong style="color:#59686b">designers</strong> and <strong style="color:#59686b">startups</strong> in UBC/SFU</h2>
-	<!--<h2 style="font-size:150px;">We're Hiring.</h2>-->
-	<!--<br><span style="font-size:45px; color:#59686b"><b>Come to our </b> IceBreaker! <a href="https://www.facebook.com/events/636441809799206/"> RVSP Here </a></span>-->
-	<!-- Begin MailChimp Signup Form -->
-	<!--<div id="mc_embed_signup">-->
-	<!--<form action="//startupstorm.us9.list-manage.com/subscribe/post?u=7800af9a0639236468185a15e&amp;id=439e2506d6" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>-->
-	<!--    <div id="mc_embed_signup_scroll">-->
-	<!--	<label for="mce-EMAIL">Keep in touch</label>-->
-	<!--	<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>-->
-	    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-	<!--    <div style="position: absolute; left: -5000px;"><input type="text" name="b_7800af9a0639236468185a15e_439e2506d6" tabindex="-1" value=""></div>-->
-	<!--    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>-->
-	<!--    </div>-->
-	<!--</form>-->
-	<!--</div>-->
-
-<!--End mc_embed_signup-->
-<!--
-	<div  class="event">
-		<h3>StormTalk #006: iOS in Swift</h3>
-		<br>
-		<span class="date">March 30th</span>
-		<span class="time">6:00PM</span>
-		<span class="location">MCLD 418, UBC Vancouver</span>
-		<br>
-		<div class="intro-button">
-			<a href="https://form.jotform.com/60878812975270">RVSP your spot.</a>
-			<a href="https://www.facebook.com/events/1587947218193620/">Facebook Event</a>
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```

Comment:
Is jQuery really necessary for just form.js? Seems like a ton of dependencies for just a few functions.